api/coverage.py

```diff
@@ -86,22 +109,11 @@ def process_item(self, identifier):
         [result] = self.process_batch([identifier])
         return result
 
-    def finalize_edition(self, edition):
-        """An OPDS entry has become an Edition. This method may (depending on
-        configuration) create a Work for that book and mark it as
+    def finalize_license_pool(self, pool):
+        """An OPDS entry has become a LicensePool. This method may (depending
+        on configuration) create a Work for that book and mark it as
         presentation-ready.
-        """
```

Comment:
All this method did was look for a LicensePool and run code on the LicensePool, so it was an easy change to make it take the LicensePool in the first place.